modules/administration-guide/partials/proc_configuring-openshift-oauth-without-initial-user.adoc

```diff
@@ -0,0 +1,38 @@
+[id="proc_configuring_openshift-oauth-without-initial-user_{context}"]
+== Configuring OpenShift OAuth without provisioning OpenShift initial OAuth user
```

Comment:
```suggestion
= Configuring OpenShift OAuth without provisioning OpenShift initial OAuth user
```

Reply:
Suggest having 1st level headings here so the headings match:
![Screenshot from 2021-02-16 09-44-15](https://user-images.githubusercontent.com/24560978/108038836-9a5b7b00-703b-11eb-83ff-991c0f2cf2dd.png)